src/LevelDBStore/Plugins/Storage/LevelDBStore.cs

```diff
@@ -9,11 +10,25 @@ public class LevelDBStore : Plugin, IStorageProvider
     {
         public override string Description => "Uses LevelDB to store the blockchain data";
 
+        private Stack<WeakReference> dbs = new();
+
         public IStore GetStore(string path)
         {
             if (Environment.CommandLine.Split(' ').Any(p => p == "/repair" || p == "--repair"))
                 DB.Repair(path, Options.Default);
-            return new Store(path);
+            IStore store = new Store(path);
```

Comment:
Dispose other references in dbs upon GetStore?

Reply:
I have no idea whether those db connections are closed or not, therefore I have to keep a weak reference of them in a stack.
In this way, I can make sure that all connections are closed in Dispose with the slightest modification to existing code.

Reply:
This method is designed for the user to be responsible for releasing the db connection.